ext/MCMCChainsMakieExt.jl

```diff
@@ -0,0 +1,74 @@
+module MCMCChainsMakieExt
+
+import MCMCChains
+import Makie
+
+
+function MCMCChains.trace(chns::T) where {T<:MCMCChains.Chains}
+    params = MCMCChains.names(chns, :parameters)
+
+    n_chains = length(MCMCChains.chains(chns))
+    n_samples = length(chns)
+    n_params = length(params)
+
+    colors = Makie.to_colormap(:tol_vibrant)
```

Comment:
As @devmotion this is a bit too opiniated, what if I want to use a different set of colors?

Reply:
I will explore ways to make more flexible.